app/controllers/admin/topics_controller.rb

```diff
@@ -366,6 +366,49 @@ def split_topic
     end
   end
 
+  def merge_tickets
+    @topics = Topic.where(id: params[:topic_ids])
+
+    @topic = Topic.new(
+      name: "MERGED: #{@topics.first.name}",
+      user: @topics.first.user,
+      forum_id: @topics.first.forum_id,
+      private: @topics.first.private,
+    )
+
+    @posts = @topic.posts
+    topics_merged = ''
+
+    if @topic.save
+      @topics.each do |t|
+        logger.info("Id #{t.id}")
+        t.posts.each do |p|
+          @posts.create(
+          body: p.body,
+          kind: p.kind,
+          user: p.user,
+          screenshots: p.screenshots,
+          attachments: p.attachments,
+          )
+        end
```

Comment:
One issue that I didn't think about in the requirements is that this generates two `Posts` with a `kind: 'first'` which causes problems with the formatting.  In this case, we should change the kind to `reply` for all `Posts` except that first one belonging to  `Topic.first`.  Hope that makes sense.